### PR TITLE
fixing cors issue

### PR DESCRIPTION
**Pull request by ram-nad**
fixes: #557 Adding the default CORS policy.

### CODE REVIEW

**Comment by fridex** [comment]
@ram-nad sorry, our CI runs on an internal network, so results of checks are not accessible outside of Red Hat (yet). Here is a log of the failing check:


```
Executing section all.python...
[33m[WARNING][0m[08:07:27] PycodestyleBear (E265): This result has no patch attached.[0m

thoth/user_api/openapi_server.py
|  54| #Add·Cross·Origin·Request·Policy·to·all
|    | [NORMAL] PycodestyleBear (E265):
|    | E265 block comment should start with '# '
Executing section all.yaml...
Executing section zuul.yaml...
Executing section cli...
```

Basically it says you should add a space after `#` sign.

Thanks for your contribution! :+1:

**Comment by ram-nad** [comment]
okk sir. I will make changes

File: Pipfile [hunk 1]
```diff
@@ -21,6 +21,7 @@ thoth-analyzer = "*"
 thoth-common = "*"
 thoth-python = "*"
 thoth-storages = "*"
+flask-cors = "*"
```
Comment (fridex) [inline Pipfile]: One minor thing, you will need to run:

```pipenv lock``` and commit also Pipfile.lock

Comment (ram-nad) [reply Pipfile]: sir i have been trying to run it. But every time it gets stuck while locking dependencies. I have left it running for hours.

Comment (fridex) [reply Pipfile]: It took few minutes on my system (5-7 mins, using Fedora:30):

```
$ pipenv --version                                                                                                                                                                                                            
pipenv, version 2018.11.26
```

Comment (ram-nad) [reply Pipfile]: Sir should i use exact same pipenv version. I am on Ubuntu 18.04.

Comment (fridex) [reply Pipfile]: You don't need to, I posted the Pipenv version in case you have some troubles to verify version I'm running.

Comment (ram-nad) [reply Pipfile]: Sir it is working now. From what I understood the issue previously was that I was trying to do pip lock without using the `flask_cors` library in my code.

**Comment by ram-nad** [comment]
> @ram-nad Nice work! thanks

Thank You, sir. I was a nice opportunity to learn. Also @goern @fridex Thanks sir for guiding me at each step.

**Comment by goern** [comment]
@ram-nad feel free so pick more issues from our projects, it was a good piece of work you did!

**Pull request merged**